RELEASE_NOTES/2.0.0-internal.6.4.0.md

```diff
@@ -0,0 +1,54 @@
+# Fluid Framework v2.0.0-internal.6.4.0
+
+Note: For the most updated release notes, see the release on GitHub at
+<https://github.com/microsoft/FluidFramework/releases/tag/client_v2.0.0-internal.6.4.0>.
```

Comment:
I don't know if we should commit this or not. It is nice to have consolidated release notes in the repo, but the "source of truth" for a given release really needs to be the GitHub release, because otherwise we have to make updates to both the GitHub release and the committed files if there are updates to the release notes.

Reply:
If there's any manual post-processing you've done relative to what the release tool does to concatenate the changesets, then yes commit it.  And consider a follow-up step to delete it along with all the changesets.  Not sure how that process works but if it's just "delete all from `.changeset` dir, then just put this in that dir for now?

Reply:
> delete it along with all the changesets

That's a good idea; riffing on it a bit, I think I'll leave the file but just make it contain a link to the source-of-truth release notes.